aws-xray-recorder-sdk-core/src/test/java/com/amazonaws/xray/contexts/LambdaSegmentContextTest.java

```diff
@@ -59,55 +58,71 @@ public void setupAWSXRay() {
     }
 
     @Test
-    public void testBeginSubsegmentWithNullTraceHeaderEnvironmentVariableResultsInADummySegmentParent() {
-        testMockContext(TraceHeader.fromString(null), FacadeSegment.class);
+    void testBeginSubsegmentWithNullTraceHeaderEnvironmentVariableResultsInADummySegmentParent() {
```

Comment:
I know this is mostly out of scope for the PR, but the name of this test isn't right. The name says we expect a Dummy (no-op) segment whereas what we test for/actually get is a facade segment. 

I'm pretty sure contrary to the name of the test always using a facade segment is the behavior we want, and we can have a larger discussion if anyone on team disagrees, but for now renaming the test to avoid future confusion would be nice.

Reply:
Thanks - yeah good cleanup